pkg/common/http.go

```diff
@@ -64,7 +64,7 @@ func HTTPResponseBody(httpResponse *http.Response) ([]byte, error) {
 
 	// Pull out response body
 	defer httpResponse.Body.Close()
-	httpResponseBodyBytes, err := ioutil.ReadAll(httpResponse.Body)
+	httpResponseBodyBytes, err := io.ReadAll(httpResponse.Body)

```

Comment:
It's a good idea to wrap this with a LimitReader first to prevent unlimited bytes being read.